recipe/meta.yaml

```diff
@@ -24,9 +19,6 @@ requirements:
     # perl is required to run the tests on UNIX.
     - perl           # [unix]
     - pkg-config     # [unix]
-    - patch          # [osx]
-    - autoconf       # [osx]
-    - automake       # [osx]
   host:
     - krb5        1.20.1
     - libnghttp2  1.52.0  # [unix]
```

Comment:
FYI, `libnghttp2 <1.57` has the [CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487). It's found recently. The fix is here https://github.com/nghttp2/nghttp2/pull/1961. 
If it's possible we should update it ASAP.

Reply:
AFAIU, this cve is related to server implementation of http2, which curl does not relies on, but I'll bump it too anyway.

Reply:
Ah, thanks for the explanation!

Reply:
`libnghttp2 1.57.0` is already on defaults